zeek/README.md

```diff
@@ -0,0 +1,189 @@
+## Overview
+
+* Zeek is the world's leading platform for network security monitoring. It interprets what it sees and creates compact, high-fidelity transaction logs, file content, and fully customized output, suitable for manual review on disk or in a more analyst-friendly tool like a security and information event management (SIEM) system.
```

Comment:
nit: I think you could include a link to [their website](https://zeek.org/)

Reply:
Done